src/sage/combinat/rigged_configurations/kleber_tree.py

```diff
@@ -122,7 +122,10 @@ def _draw_tree(tree_node, node_label=True, style_point=None, style_node='fill=wh
         start = [0., 0.]
     if rpos is None:
         rpos = [0., 0.]
-    draw_point = lambda point: '(%.3f, %.3f)' % (point[0],point[1])
+
+    def draw_point(point):
+        return '(%.3f, %.3f)' % (point[0], point[1])
```

Comment:
Unless I'm missing something, it looks like `draw_point` is only used once? Can't this string formatting just be done in that line? I don't think this one line function that's used in a single place is useful for readability. And at worst it adds (marginal) overhead having to call a separate function for a simple string formatting in a line where we're already doing string formatting.

Reply:
done